### ISSUE TITLE
Color of dead links: contrast issue

### ISSUE THREAD

**Issue by estelle** [mdn/yari#5906]
### Summary

When a page isn't written yet, the link is red based on the class `.page-not-yet-written`.
The HTML section has red links.
The specificity of those links overrides the colors based on it not yet being written.
There isn't enough differentiation between the two reds to see that the "not yet written" links are, well, not yet written, if those links got the right red.

### URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/max

### Reproduction steps

1. visit https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/max
2. in the see also section, `low`, `optimimum` and `high` are dead links. 
3. 

### Expected behavior

links with the class `.page-not-yet-written` should be a different color from other links

### Actual behavior

they're all the same color

### Device

Desktop

### Browser

Firefox

### Browser version

Stable

### Operating system

Mac OS

### Anything else?

this could be solved with a more specific select.

```css
a.page-not-yet-written.page-not-yet-written.page-not-yet-written {
/* this has a specificity of 0-3-1 */
}
```

### Validations

- [X] I have read the [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
- [X] I have verified that there isn't [already an issue](https://github.com/mdn/yari/issues) that reports the same bug to avoid creating a duplicate.
- [X] I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mdn/yari/discussions).

**Comment by caugner** [comment]
I wonder whether a `text-decoration-style: wavy` wouldn't even be better than a slightly different color?

<img width="382" alt="image" src="https://user-images.githubusercontent.com/495429/161730253-b4c4fc85-a602-4b34-a833-8d6290b43b2e.png">

/cc @schalkneethling

**Comment by caugner** [comment]
(Duplicate of #5430.)

**Comment by schalkneethling** [comment]
This has been addressed in the readability work here https://github.com/mdn/yari/discussions/5715

## Dark Mode

![Screenshot of page-not-created links in dark mode. Shows links as red with a wavy underline style](https://user-images.githubusercontent.com/10350960/162009857-3fb69c27-b764-445d-b577-d253acc784b0.png)

## Light mode

![Screenshot of page-not-created links in light mode. Shows links as red with a wavy underline style](https://user-images.githubusercontent.com/10350960/162010364-4d34c092-434c-46b7-a0c9-ea1dff0ef7f5.png)
